cvmfs/cvmfs.cc

```diff
@@ -1178,6 +1199,8 @@ static void cvmfs_statfs(fuse_req_t req, fuse_ino_t ino) {
   struct statvfs info;
   memset(&info, 0, sizeof(info));
 
+  TraceInode(Tracer::kEventStat, ino, "statfs()");
```

Comment:
We should change the constant to `kEventStatFs` to avoid confusion with the stating of files.